src/metabase/driver.clj

```diff
@@ -405,6 +405,11 @@
     ;; subselects in SQL queries.
     :nested-queries
 
+    ;; Does the driver support persisting models
+    :persist-models
+    ;; Is persisting enabled?
+    :persist-models-enabled
```

Comment:
In what situation would knowing the driver supports persisting models but it's **not** enabled be useful? Can't these be combined into one feature?

Reply:
Persistence is only offered on databases that a) support schemas and b) that we have built the necessary ddl functionality. Thus we need to know which databases are eligible for persistence (`(supports? :persist-models)`) and which have that functionality enabled and setup (`(supports? :persist-models-enabled)`).

Reply:
Doesn't the `enabled-persisted-models` track whether this is enabled or not? Why would we need both a driver feature flag and a Setting?

Reply:
Hmmm. I think `persist-models-enabled` makes more sense as a [Database-local Setting](https://github.com/metabase/metabase/pull/19399) rather than a "does the driver support this for this version of the DB?" feature flag